{
    "title": "A flaw was found in Ansible if an ansible user sets ANSIBLE_ASYNC_DIR to a subdirectory of a world writable directory. When this occurs, there is a race condition on the managed machine. A malicious, non-privileged account on the remote machine can exploit the race condition to access the async result data. This flaw affects Ansible Tower 3.7 and Ansible Automation Platform 1.2.",
    "score": 2.5,
    "severity": "LOW",
    "cwe_id": "CWE-362",
    "cvss_vector": "CVSS:3.1/AV:L/AC:H/PR:L/UI:N/S:U/C:L/I:N/A:N",
    "cpe_match": [
        "cpe:2.3:a:redhat:ansible_tower:3.7.0:*:*:*:*:*:*:*",
        "cpe:2.3:a:redhat:ansible_automation_platform:1.2:*:*:*:*:*:*:*",
        "cpe:2.3:o:redhat:enterprise_linux:7.0:*:*:*:*:*:*:*",
        "cpe:2.3:a:redhat:ansible_engine:2.0:*:*:*:*:*:*:*",
        "cpe:2.3:a:redhat:ansible_tower:3.0:*:*:*:*:*:*:*",
        "cpe:2.3:o:fedoraproject:fedora:34:*:*:*:*:*:*:*",
        "cpe:2.3:a:redhat:openstack-rdo:-:*:*:*:*:*:*:*"
    ],
    "cve_source": [
        "NVD"
    ],
    "ttpMapping": [
        {
            "technique_id": "T1083",
            "confidence": "10.4",
            "technique_name": "File and Directory Discovery"
        },
        {
            "technique_id": "T1057",
            "confidence": "10.3",
            "technique_name": "Process Discovery"
        },
        {
            "technique_id": "T1083",
            "confidence": "11.7",
            "technique_name": "File and Directory Discovery"
        }
    ]
}
